app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html

```diff
@@ -1,4 +1,4 @@
-<form name="form" class="container-fluid form-horizontal">
+<form name="form" class="container-fluid form-horizontal" style="padding: 0;">
```

Comment:
This fixes this issue:
Before:
![wizard_before](https://cloud.githubusercontent.com/assets/13868700/19936379/9301d7c6-a0f3-11e6-8645-41257d1edbf2.png)

After:
![wizard_after](https://cloud.githubusercontent.com/assets/13868700/19936384/997788a8-a0f3-11e6-8339-485d85abdd98.png)